requirements-core.txt

```diff
@@ -22,7 +22,7 @@ python-louvain>=0.13
 requests
 openTSNE>=0.6.1
 baycomp>=1.0.2
-pandas>=1.3.0,!=1.5.0
+pandas>=1.4.0,!=1.5.0
```

Comment:
This should be changed in meta.yaml too.

Reply:
Done.